docs/GooglePay.md

```diff
@@ -0,0 +1,176 @@
+
+## Setup Stripe
+
+### Server side
+
+//
+
+### Client side
+
+The [React Native SDK](https://github.com/stripe/stripe-react-native) is open source and fully documented. Internally, it makes use of native iOS and Android SDKs.
+Install the SDK by running:
+
+```sh
+yarn add react-native-stripe-sdk
+```
+
+For iOS you will have to run `pod install` inside `ios` directory in order to install needed native dependencies. Android won't require any additional steps.
+
+When your app starts, configure the SDK with your Stripe [publishable key](https://stripe.com/dashboard.stripe.com/account/apikeys) so it can make requests to the Stripe API:
+
+```tsx
+import { StripeProvider } from '@stripe/stripe-react-native';
+
+function App() {
+  return (
+    <StripeProvider
+      publishableKey="pk_test_51ITUqcBDuqlYGNW28TM0Of8JNHxKgXKlBOzNi0sILDmyqxsCd6ACZmb2cjstrcrfnhXJR47w9D8Tezqbdqm24HRQ00lqUzh0eg"
+      urlScheme="your-url-scheme" // required for 3D Secure and bank redirects
+      merchantIdentifier="merchant.com.{{YOUR_APP_NAME}}" // required for Apple Pay
+    >
+      // Your app code here
+    </StripeProvider>
+  );
+}
+```
+
+## Initialize Google Pay
+
+Next, initialize a Google Pay by calling `initGooglePay` method. Call this on initial render of your checkout screen for best performance.
+
+`initGooglePay` accepts several configuration options that can be customized for your needs.
+See [SDK Reference](https://stripe.dev/stripe-react-native/api-reference/index.html) for more details.
+
+```tsx
+import { useGooglePay } from 'stripe-react-native';
+
+function CheckoutScreen() {
+  const { initGooglePay } = useGooglePay();
+  const [initialized, setInitialized] = useState(false)
+
+  useEffect(() => {
+    async function initialize() {
+      const { error } = await initGooglePay({
+        testEnv: true,
+        merchantName: 'Widget Store'
+        countryCode: 'US',
+        billingAddressConfig: {
+          format: 'FULL',
+          isPhoneNumberRequired: true,
+          isRequired: false,
+        },
+        existingPaymentMethodRequired: false,
+        isEmailRequired: true,
+      });
+
+      if (error) {
+        Alert.alert(error.code, error.message);
+        return;
+      }
+      setInitialized(true);
+    }
+    initialize();
+  }, [initGooglePay]);
+}
+```
+
+After initializing GooglePay, the `initGooglePay` promise is resolved, it's successful if there is no error. This information can be used to update your UI to indicate to your customer that Google Pay is ready to be used.
+
+```tsx
+import { useGooglePay } from 'stripe-react-native';
+
+function CheckoutScreen() {
+  const [initialized, setInitialized] = useState(false);
+
+  // continued from above
+
+  return (
+    <Screen>
+      <Button
```

Comment:
Let's link to https://developers.google.com/pay/api/android/guides/brand-guidelines for reference on how to style the button